goto/gotomagic/magic.py

```diff
@@ -61,6 +62,19 @@ def update_shortcut(self, magicword, uri):
             )
             exit(1)
 
+    def rename_shortcut(self, from_magicword, to_magicword, overwrite=False):
+        """ Renaming a shortcut """
+        if not overwrite and to_magicword in self.magic:
+            uri = self.magic[to_magicword]
+            return GotoWarning('adding_existing_magicword_short', magicword=to_magicword, uri=uri)  # noqa
```

Comment:
What does #noqa mean?

Reply:
no quality assurrance. It effictively turns off pep8 linting on that mine. making that line be an explicit exemption of the pep8 rules. the line is too long. thats why. 

If you enable pep8 linting in your editor, you get warnings when the code style breaks pep8 style.